lib/absmultimarcas.rb

```diff
@@ -17,7 +17,27 @@ def self.all
   end
   class F1SalesCustom::Email::Parser
     def parse
-      # ...
+      parsed_email = @email.body.colons_to_hash
+      source = F1SalesCustom::Email::Source.all[0]
+      product = "#{parsed_email['marca'].capitalize} #{parsed_email['modelo'].capitalize}"
+      description = "Versão: #{parsed_email['versao']} - Ano: #{parsed_email['ano']} - Valor: #{parsed_email['preco']}"
+
+      {
+        source: {
+          name: source[:name]
+        },
+        customer: {
+          name: parsed_email['nome_completo'],
+          phone: parsed_email['celular'].tr('^0-9', ''),
+          email: parsed_email['email']
+        },
+        observation: {
+          text: parsed_email['melhor_forma_de_iniciar_a_negociacao'].gsub('_', ' ')
```

Comment:
Is that the correct field to put this information?

Examples of email info that I'm putting here:
`Trocar_meu_veículo_por_um_novo_e_ainda_receber_um_valor_em_dinheiro` 
`Quero_comprar_com_zero_de_entrada_100%_financiado `